### ISSUE TITLE
Team name conflict with user, user deleted, but still cannot create Team

### ISSUE THREAD

**Issue by ramarnat** [keybase/keybase-issues#3207]
I deleted the user conflicting with the team name, but it still does not allow the team to be created

**Comment by mlsteele** [comment]
That is expected. A team can't have the same name as a deleted user. Teams and users share the same namespace and deleting a user makes that name unusable. I hope this isn't too frustrating! Were there any points along the way where you felt mislead by warning or errors?

**Comment by ramarnat** [comment]
any reason why? I understand not having teams and users having the same namespace, but we created a company user way before teams were created, and now cannot use that as the basis for the team name.

**Comment by malgorithms** [comment]
We've made 2 separate design decisions, which come together to affect your question:

1. a name on keybase cannot be repurposed. (If I delete `chris` no one new can register it)
2. team names and usernames can't currently coexist (`chris` can't be both a team and a user)

you can imagine how, logically, they end up combining into the idea, that `chris`, in keybase's view, is either a team or a user, and that team or user happens to be deleted or not. So when you try to register `chris` the big question is whether it has _ever_ been used or not....there isn't some specific exception of it happens to be BOTH the opposite kind of account AND it's deleted. Instead we simply prevent it.

This is a UX decision, really. It's solvable, technically, but consider all the problems that happen if everywhere a team or user is mentioned, it's necessary to clarify which it is in order to protect a user from confusion. Right now I can go do this in the command line:

```
keybase chat send foobar -m "hi friend"
```

Let's say `foobar` is someone I know. But then she deletes her account and then you, malicious user, come along and create the team `foobar` and add me to it. If I run that exact same command, it will deliver the message into the team chat.

That's just one example. Another is that simple team chats and user chats are comingled in the inbox, and we're going to allow teams to have avatars soon. So someone could try to be tricky by claiming up team names of everyone who deletes their account.

In other words, we have the goal of identifying a cryptographic chain with a single identifier....and we lose that if we allow re-claiming by others after deletion.

Ultimately we could work our way across all these problems...by say introducing the concept of "converting" a user to a team through some signed statement. It's just not a small task. And if there's some possible point of confusion we haven't thought of, we might end up regretting allowing it. Because once we support such a thing, we can't change our minds later on and we're stuck backwards supporting it for all time.

So, all said, it's unlikely to happen soon.

**Comment by ramarnat** [comment]
I agree that conversion is the right approach. Our keybase id - full360 was verified with our twitter and github presence. We have had this situation before (joys of being early adopters), both github and docker allowed for the conversion of an account to an "organization". 

Now, unfortunately, I am stuck without a team or a user with our org name. So that is a bit frustrating.

**Comment by maduce** [comment]
Having the same exact issue. Made a user with the name and didn't know that a team with the same name couldnt be made. Wishing I could change the username and make the team name. Is it possible to have two word team names or use a dash, i.e., acme-team?

**Comment by akshaykhurana** [comment]
Similar issue at our organisation. Bumping this thread.
